Flow.Launcher.Core/Plugin/JsonRPCPlugin.cs

```diff
@@ -82,12 +88,15 @@ public List<Result> LoadContextMenus(Result selectedResult)
         };
         private Dictionary<string, object> Settings { get; set; }
 
-        private Dictionary<string, FrameworkElement> _settingControls = new();
+        private readonly Dictionary<string, FrameworkElement> _settingControls = new();
 
         private async Task<List<Result>> DeserializedResultAsync(Stream output)
         {
             if (output == Stream.Null) return null;
 
+            var temp = System.Text.Encoding.UTF8.GetString(((MemoryStream)output).ToArray());
+            System.Diagnostics.Debug.WriteLine(temp);
```

Comment:
Is this accidentally added for debugging?

Reply:
good catch, removed other debugging, but missed this one. Thanks!